include/openspace/interaction/joystickcamerastates.h

```diff
@@ -49,17 +49,22 @@ class JoystickCameraStates : public CameraInteractionStates {
         GlobalRollX,
         GlobalRollY,
         PanX,
-        PanY
+        PanY,
+        Property
+    };
+
+    enum class JoystickType {
+        JoystickLike = 0,
```

Comment:
No need to mark the first one as 0 since we don't use its numerical value anywhere